.travis.yml

```diff
@@ -3,7 +3,7 @@ rust:
 - stable
 - beta
 - nightly
-- 1.5.0
+- 1.6.0
```

Comment:
This might require a major bump unfortunately. I never got around to figuring out a good support story for older compilers, but someone did ask me to support 1.5 in serde (or syntex? I can't remember) but either way I don't want to break people that might depend on this.